### ISSUE TITLE
ElkanotoPuClassifier not working with the scikit-learn interface of xgboost

### ISSUE THREAD

**Issue by shaypal5** [pulearn/pulearn#20]
@asitstands wrote:

Hi, I'm trying to use ElkanotoPuClassifier with the [scikit-learn interface of xgboost](https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.XGBClassifier). It fails with an error saying that labels should be 0 or 1. Marking unlabeled as -1 does not work with xgboost. Is there any workaround?
